contrib/generate_precompile.jl

```diff
@@ -183,10 +183,10 @@ for match = Base._methods(+, (Int, Int), -1, Base.get_world_counter())
     # interactive startup uses this
     write(IOBuffer(), "")
 
-    # not critical, but helps hide unrelated compilation from @time when using --trace-compile
-    foo() = rand(2,2) * rand(2,2)
-    @time foo()
-    @time foo()
+    # Not critical, but helps hide unrelated compilation from @time when using --trace-compile.
+    f55729() = Base.Experimental.@force_compile
```

Comment:
This PR works, but probably not for the reasons you intended. You're trying to using the local variable `f55729` as a global, causing nothing to happen:
```UndefVarError: `f55729` not defined in `Main```